apkid/rules/elf/packers.yara

```diff
@@ -129,11 +129,11 @@ rule upx_elf_3_09 : packer
   meta:
     description = "UPX 3.09 (unmodified)"
 
-    strings:
```

Comment:
In the future, try and put white space changes in a separate PR or at least a commit. Also, these aren't consistent.